### ISSUE TITLE
App throws IllegalParameterValueException

### ISSUE THREAD

**Issue by TanyaEf** [Jaspersoft/jrsh#173]
### Export operation

If you'll miss a slash before repository path and press `Tab` key for completion, the app will crash with `IllegalParameterValueException` exception:
```bash
$> export repository pub `press Tab key`
 Exception in thread "main" com.jaspersoft.jasperserver.jaxrs.client.core.exceptions.IllegalParameterValueException: The value "pub" for parameter "folderUri" is invalid.
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:57)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:526)
        at com.jaspersoft.jasperserver.jaxrs.client.core.exceptions.handling.DefaultErrorHandler.handleBodyError(DefaultErrorHandler.java:95)
        at com.jaspersoft.jasperserver.jaxrs.client.core.exceptions.handling.DefaultErrorHandler.handleError(DefaultErrorHandler.java:67)
        at com.jaspersoft.jasperserver.jaxrs.client.core.JerseyRequest.executeRequest(JerseyRequest.java:142)
        at com.jaspersoft.jasperserver.jaxrs.client.core.JerseyRequest.executeRequest(JerseyRequest.java:130)
        at com.jaspersoft.jasperserver.jaxrs.client.core.JerseyRequest.get(JerseyRequest.java:99)
        at com.jaspersoft.jasperserver.jaxrs.client.apiadapters.resources.BatchResourcesAdapter.search(BatchResourcesAdapter.java:48)
        at com.jaspersoft.jasperserver.jrsh.core.completion.impl.RepositoryCompleter$Downloader.download(RepositoryCompleter.java:285)
        at com.jaspersoft.jasperserver.jrsh.core.completion.impl.RepositoryCompleter.complete(RepositoryCompleter.java:66)
        at com.jaspersoft.jasperserver.jrsh.core.completion.impl.RepositoryCompleter.complete(RepositoryCompleter.java:106)
        at jline.console.completer.ArgumentCompleter.complete(ArgumentCompleter.java:152)
        at jline.console.completer.AggregateCompleter$Completion.complete(AggregateCompleter.java:121)
        at jline.console.completer.AggregateCompleter.complete(AggregateCompleter.java:80)
        at jline.console.ConsoleReader.complete(ConsoleReader.java:3261)
        at jline.console.ConsoleReader.readLine(ConsoleReader.java:2621)
        at jline.console.ConsoleReader.readLine(ConsoleReader.java:2269)
        at jline.console.ConsoleReader.readLine(ConsoleReader.java:2257)
        at com.jaspersoft.jasperserver.jrsh.core.evaluation.strategy.impl.ShellEvaluationStrategy.eval(ShellEvaluationStrategy.java:50)
        at com.jaspersoft.jasperserver.jrsh.runner.App.main(App.java:17)
```
We should handle this exception and let JRSH continue REPL. Any wrong path should be ignored by `RepositoryCompleter`.

**Comment by Krasnyanskiy** [comment]
Yup, it's a bug.

**Comment by Krasnyanskiy** [comment]
Possible duplicate of #164.

**Comment by Krasnyanskiy** [comment]
Fixed in [v2.0.6](https://github.com/Jaspersoft/jrsh/releases/tag/v2.0.6)
